pyproject.toml

```diff
@@ -2,6 +2,126 @@
 requires = ["setuptools>=64", "setuptools_scm[toml]>=8"]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "ctapipe"
+description = "Prototype CTA event reconstruction pipeline"
```

Comment:
maybe now is a good idea to update this? Perhaps:

```toml
description = "Event reconstruction framework for Imaging Atmospheric Cherenkov Telescopes"
```

Reply:
if we want to be more ambitious, could add something about it being the official event reconstruction framework for the CTA Observatory.

Reply:
done